src/WebUI/Common/KubeSummary.tsx

```diff
@@ -94,6 +99,7 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
         this._selectionStore.addChangedListener(this._onSelectionStoreChanged);
 
         this._workloadsActionCreator = ActionsCreatorManager.GetActionCreator<WorkloadsActionsCreator>(WorkloadsActionsCreator);
+        this._selectionActionCreator = ActionsCreatorManager.GetActionCreator<SelectionActionsCreator>(SelectionActionsCreator);
```

Comment:
Should this be ImageActionsCreator instead?

Reply:
No..The selection action is required when clicking on Back button of ImageDetails view